src/Core/src/ViewExtensions.cs

```diff
@@ -48,27 +48,30 @@ public static partial class ViewExtensions
 			await Screenshot.Default.CaptureAsync(window);
 #endif
 
-#if !TIZEN
-		internal static bool NeedsContainer(this IView? view)
+		internal static bool NeedsContainer(this IView? view, PlatformView? platformView)
 		{
+#if !TIZEN
 			if (view?.Clip != null || view?.Shadow != null)
 				return true;
+#endif
 
 #if ANDROID
-			if (view?.InputTransparent == true)
+			if (view?.InputTransparent == true && (platformView is null || platformView is not IInputTransparentManagingView))
```

Comment:
This line matches the change in the transparency logic - if we can handle it ourselves, we don't wrap.